Makefile

```diff
@@ -160,6 +160,9 @@ fluentd:
 image-cleanup-test-images:
 	$(MAKE) -C misc/image-cleanup-test-images $(MFLAGS)
 
+functional-tests-image:
```

Comment:
can you rename this to something more descriptive? Example: `ecr-execution-role-image`